ui/packages/consul-ui/app/templates/dc/kv/index.hbs

```diff
@@ -4,6 +4,7 @@
 ) as |filters|}}
   {{#let (or sortBy "Kind:asc") as |sort|}}
     <AppView>
+    {{#if (not-eq parent.Key '/') }}
```

Comment:
Why is this line repeated again in line 10?

Reply:
🤔 good point! Lemme change up.